components/support/support.jsx

```diff
@@ -4,25 +4,34 @@ import './support-style';
 
 export default class Support extends React.Component {
   render() {
-    let { number, type } = this.props;
+    let { minimum, maximum, size, type } = this.props;
```

Comment:
I might rename the `size` prop to `level` or `rank` or something similar. "Size" sounds more like count to display or image size to me than their rank based on donation amount. Also, maybe instead of `minimum` and `maximum` being props along with `level`, we could have that data within the component file so only `level` needs to be passed.